packages/server/test/react.test.tsx

```diff
@@ -389,6 +399,90 @@ test('mutation on mount + subscribe for it', async () => {
   });
 });
 
+describe('useMutation()', () => {
```

Comment:
will probably need to rewrite the test cases later to allow for testing the hooks in isolation instead of rendering